maven-plugin-plugin/src/it/mplugin-370-maven-deps-scope-good/pom.xml

```diff
@@ -38,6 +38,12 @@ under the License.
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>org.apache.maven</groupId>
+      <artifactId>maven-jxr</artifactId>
+      <version>3.1.1</version>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
Should be also added `org.apache.maven:maven-archiver`  unless is present on dependency tree

Reply:
We have identified these two "cuckoo eggs" but am using only jxr in IT as maven-archiver pulls in a bunch of other deps, making IT run eons. I think for IT using jxr only is okay .